docs/community/community-overview.md

```diff
@@ -1,17 +1,27 @@
-
-Our community can usually be found hanging out on [Discord](https://discord.umaproject.org/),  where you can find details on upcoming proposals and votes, get support with using UMA tokens and products as well as exploring new ideas and how we can turn them into reality. 
-
-We also have a [Discourse](https://discourse.umaproject.org/), where improvements to our protocol are discussed as well as proposing ideas for new products. We also maintain a forum where UMA community members can [showcase their talents](https://discourse.umaproject.org/c/finding-people/talent-available/24), and product teams can [offer bounties](https://discourse.umaproject.org/c/finding-people/opportunities-at-uma-collaborators/22) to our community for collaborative working. We  have a number of [outstanding vacancies](https://angel.co/company/uma-project) as well as a [“talent referral options”](https://medium.com/uma-project/talent-referral-options-program-170bc347542a) program, which offers up to 1000 $UMA to anyone who refers suitably qualified candidates for any of our open positions.
-
-You can also keep up to date with what is happening in UMA by following our [Twitter](https://twitter.com/UMAprotocol); longer form explorations of UMA developments are available on our [Medium](https://medium.com/uma-project), while our [YouTube](https://www.youtube.com/channel/UC-3qS7FXxCd7gBMLttmTirw/playlists) channel hosts presentations and tutorials in video format.  There is an ongoing [Coinbase Earn](https://www.coinbase.com/earn/uma) campaign which gives a video overview of UMA as well the opportunity to collect a few tokens for watching explainer videos to allow you to get up to speed with one of the most innovative and fast moving protocols in the DeFi space.
-
-
-
-
-
+The UMA community consists of tokenholders, financial engineers, product designers, developers, and users. We're happy to have you join and always willing to help people who have just discovered UMA make the most out of our innovations. 
+
+### Community Links
+You can keep up to date with what is happening with UMA across our various social media platforms and community resources:
+- [Discord](https://discord.umaproject.org/)
+- [Twitter](https://twitter.com/UMAprotocol)
+- [Discourse](https://discourse.umaproject.org/)
+- [Medium](https://medium.com/uma-project)
+- [YouTube](https://www.youtube.com/channel/UC-3qS7FXxCd7gBMLttmTirw/playlists)
+- [UMAverse](https://projects.umaproject.org/)
+- [UMA Protocol Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_soder0b7n0mgutr5jdbin9aqgs@group.calendar.google.com&ctz)
+
+### SuperUMAn Resources
+Below are SumperUMAn specific resources if you are interested in becoming a SuperUMAn:
+- [SuperUMAn Discord](https://discord.gg/WaSdQwJu)
+- [SuperUMAn Twitter](https://twitter.com/SuperUmans)
+- [Becoming a SuperUMAn](/community/superUMAns)
```

Comment:
It'd be better to make this linked title match sidebar + the title on the page that is linked to.